extensions/blocks/simple-payments/utils.js

```diff
@@ -21,9 +22,50 @@ export const decimalPlaces = number => {
 	return Math.max( 0, ( match[ 1 ] ? match[ 1 ].length : 0 ) - ( match[ 2 ] ? +match[ 2 ] : 0 ) );
 };
 
-export const formatPrice = ( price, currency, withSymbol = true ) => {
+const getNavigatorLanguage = () => {
+	if ( navigator.languages && navigator.languages.length ) {
+		return navigator.languages[ 0 ];
+	}
+	return navigator.userLanguage || navigator.language || navigator.browserLanguage || 'en';
+};
```

Comment:
I grabbed this after looking through https://stackoverflow.com/questions/673905/best-way-to-determine-users-locale-within-browser .

Reply:
Wouldn't you need it in site's locale, rather than user's browser? 🤔 I might be missing some context here tho. :-)

Reply:
Yeah.. we're trying to find the best way to get the site's locale.  I've removed a couple of methods that don't work and the navigator locale is all that's left right now.  I'll add `[WIP]` to the PR to be a bit clearer